app/src/main/java/edu/uc/coffeens/stocktracker/services/RetrofitClientInstance.kt

```diff
@@ -9,7 +9,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://gist.githubusercontent.com/"
+    private const val BASE_URL = "https://gist.githubusercontent.com/"
```

Comment:
In an attempt to make sure the code's modifiers were descriptive as possible, I added the const modifier as specified in here https://kotlinlang.org/docs/coding-conventions.html#property-names